partials/tabs/projects.html

```diff
@@ -96,7 +87,7 @@
       opacity: 0.8, // Opacity of modal background
       in_duration: 300, // Transition in duration
       out_duration: 200, // Transition out duration
-      starting_top: '50%', // Starting top style attribute
+      starting_top: '10%', // Starting top style attribute
```

Comment:
linter integration project's popup was very large and was not fitting in the window, therefore this shifts the box a little up.

Before
![screen shot 2017-01-19 at 1 28 23 am](https://cloud.githubusercontent.com/assets/13018570/22080727/9e53395a-dde6-11e6-9a62-232b9bb23b53.png)
After

![screen shot 2017-01-19 at 1 28 07 am](https://cloud.githubusercontent.com/assets/13018570/22080735/a58ddbda-dde6-11e6-82a9-a45fb2868ca1.png)